docs/index.rst

```diff
@@ -125,39 +100,141 @@ argument.
    :maxdepth: 2
 
 
-Fast conversion between Date and CXC seconds
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Fast conversion between formats
+-------------------------------
+
+Converting between time formats (e.g. from CXC seconds to Year Day-of-Year) is
+easily done with the |CxoTime| class, but this involves some overhead and is
+relatively slow for scalar values or small arrays (less than around 100
+elements). For applications where this conversion time ends up being significant,
+the `cxotime` package provides a different interface that is typically at least
+10x faster for scalar values or small arrays.
 
-For fast conversion of an input date or dates in Year Day-of-Year date format,
-the function :func:`~cxotime.cxotime.date2secs` can be used.
+For fast conversion of an input date or dates to a different format there are
+two options that are described in the next two sections.
 
-This is a specialized function similar to the legacy ``Chandra.Time.date2secs``
-that allows for fast conversion of a single date or an array of dates to CXC
-seconds.  It is intended to be used ONLY when the input date is known to be in
-the correct Year Day-of-Year format.
+``convert_time_format``
+^^^^^^^^^^^^^^^^^^^^^^^
 
-The main use case is for a single date or a few dates. For a single date this
-function is about 10 times faster than the equivalent call to
-``CxoTime(date).secs``. For a large array of dates (more than about 100) this
-function  is not significantly faster.
+The first option is a generalized time format conversion function
+:func:`~cxotime.convert.convert_time_format` that can be used to convert between
+any of the supported *fast* formats:
 
-This function will raise an exception if the input date is not in one of
-these allowed formats:
+- `secs`: CXC seconds
+- `date`: Year Day-of-Year
+- `greta`: GRETA format (input can be string, float or int)
+- `maude`: MAUDE format (input can be string or int)
+- `jd`: Julian Day (requires `fmt_in="jd"` to identity this format)
 
-- YYYY:DDD
-- YYYY:DDD:HH:MM
-- YYYY:DDD:HH:MM:SS
-- YYYY:DDD:HH:MM:SS.sss
+For example::
+
+  >>> from cxotime import convert_time_format
+  >>> convert_time_format("2022:001:00:00:00.123", "greta")
+  '2022001.000000123'
+  >>> convert_time_format(100.123, "date")
+ '1998:001:00:00:36.939'
```

Comment:
I think this line screws up the alignment.

I think it is better to have four spaces in all anyway, like the code below.